src/utils/masks.js

```diff
@@ -0,0 +1,31 @@
+import createNumberMask from 'text-mask-addons/dist/createNumberMask';
+import emailMask from 'text-mask-addons/dist/emailMask';
+
+export const MASKS = {
```

Comment:
Can we expose the existing REGEX masks we seem to copy/paste from project to project?
This way we ensure we expose masks that already meet the business's needs.
https://bitbucket.org/search?q=validationhelpers.js&account=%7Bf0e46ff7-117d-4015-9027-923794846585%7D

Reply:
@SeanGroff  Seems like those regex are formatted different from what `react-text-mask` expects. Should I gather the ones that are common across all those files and create an equivalent of them for `react-text-mask`, or are you suggesting we pull in all of that validation logic as well?

Reply:
> @SeanGroff Seems like those regex are formatted different from what `react-text-mask` expects. Should I gather the ones that are common across all those files and create an equivalent of them for `react-text-mask`, or are you suggesting we pull in all of that validation logic as well?

Gather the ones that are common across all those files and create an equivalent of them for `react-text-mask` would be ideal! I haven't looked at react-text-mask in a long time so I don't have a better suggestion.

I just want to ensure how we validate a phone or a zipcode remains the same as whats in production for several apps currently.

Reply:
Gotcha. I'll work on that